### PR TITLE
[Security] Mention customizing successful and failed authentication

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/4258, part of https://github.com/symfony/symfony-docs/issues/15908

### CODE REVIEW

File: security.rst [hunk 1]
```diff
@@ -1602,6 +1602,20 @@ and set the ``limiter`` option to its service ID:
             ;
         };
 
+Customize Successful And Failed Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (alamirault) [inline security.rst]: Should we add something like "behavior" / "handling"

`Customize Successful And Failed Authentication Behavior`
```

Comment (alexandre-daubois) [reply security.rst]: Good idea, updated :slightly_smiling_face:

**Review by alamirault — APPROVED** [review]

File: security.rst [hunk 2]
```diff
@@ -1602,6 +1602,20 @@ and set the ``limiter`` option to its service ID:
             ;
         };
 
+Customize Successful And Failed Authentication Behavior
```
Comment (OskarStark) [inline security.rst]:
```suggestion
Customize Successful and Failed Authentication Behavior
```

Comment (alexandre-daubois) [reply security.rst]: Updated :slightly_smiling_face:

**Comment by javiereguiluz** [comment]
Alex, you are amazing! You solved one of the oldest pending issue in the docs 🙇  _(it was pending since 2014 !!)_ Thanks!

**Pull request merged**